### PR TITLE
Make Netty SO_BACKLOG configurable

### PR DESCRIPTION
**Pull request by kangkaisen**
For https://github.com/apache/incubator-doris/issues/1637

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -215,6 +215,14 @@
      * Currently, all FEs' http port must be same.
      */
     @ConfField public static int http_port = 8030;
+
+    /*
+     * The backlog_num for netty http server
+     * When you enlarge this backlog_num, you should ensure it's value larger than
+     * the linux /proc/sys/net/core/somaxconn config
+     */
+    @ConfField public static int http_backlog_num = 128;
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/common/Config.java]:
```suggestion
    @ConfField public static int http_backlog_num = 1024;
```

**Review by imay — COMMENTED** [review]
Besides this, thrift server also have a default backlog(50).
It's better if you can modify it together.

**Comment by kangkaisen** [comment]
> Besides this, thrift server also have a default backlog(50).
> It's better if you can modify it together.

OK, Update.

File: fe/src/main/java/org/apache/doris/common/Config.java (hunk 1 above)
Comment (kangkaisen) [reply fe/src/main/java/org/apache/doris/common/Config.java]: update

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**